build/taskctl/tasks.yaml

```diff
@@ -47,19 +47,22 @@ tasks:
     context: powershell
     description: Perform Terraform validation
     command:
-      - Invoke-Terraform -Validate -Path $env:TF_FILE_LOCATION
+      - ln -sfn /usr/local/terraform/1.9.3/bin /usr/local/terraform/bin;
```

Comment:
If you use `ensono/eir-infrastructure:1.1.214` then you can drop a file in the root of the repo `.terraform-version` with the version number you want, the container has `1.9.3`.